packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewerContainer.spec.js

```diff
@@ -1,199 +1,70 @@
-  locations: [
-    { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
-  ]
-})
-
-const errorSubject = Factory.build('subject', {
-  contentLoadingState: asyncStates.error,
-  error: new Error('Error loading subject'),
-  locations: [
-    { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
-  ]
-})
-
-const initializedSubject = Factory.build('subject', {
-  contentLoadingState: asyncStates.initialized,
-  locations: [
-    { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
-  ]
-})
-
-const loadingSubject = Factory.build('subject', {
-  contentLoadingState: asyncStates.loading,
-  locations: [
-    { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
-  ]
-})
 
 describe('Component > SingleTextViewerContainer', function () {
   it('should render without crashing', function () {
-    const wrapper = mount(<SingleTextViewerContainer />)
-    expect(wrapper).to.be.ok()
-    expect(wrapper.html()).to.be.null()
+    render(
+      <SingleTextViewerContainer />
+    )
+    expect(screen).to.be.ok()
   })
 
   describe('with content loading state of error', function () {
-    let wrapper, onErrorSpy, onReadySpy
-
-    before(function () {
-      onErrorSpy = sinon.spy()
-      onReadySpy = sinon.spy()
+    it('should not render the text subject content', function () {
+      const mockError = new Error('Error loading subject')
 
-      wrapper = mount(
+      render(
         <SingleTextViewerContainer
-          content={errorSubject.content}
-          contentLoadingState={errorSubject.contentLoadingState}
-          error={errorSubject.error}
-          onError={onErrorSpy}
-          onReady={onReadySpy}
-        />)
-    })
-
-    it('should render null', function () {
-      expect(wrapper.html()).to.be.null()
-    })
+          content='test subject content'
+          contentLoadingState={asyncStates.error}
+          error={mockError}
+        />
+      )
 
-    it('should call the onError prop', function () {
-      expect(onErrorSpy).to.have.been.calledOnceWith(errorSubject.error)
-    })
-
-    it('should not call the onReady prop', function () {
-      expect(onReadySpy).to.not.have.been.called()
+      expect(screen.queryByText('test subject content')).to.be.null()
     })
   })
 
   describe('with content loading state of success', function () {
-    let wrapper, onErrorSpy, onReadySpy
-
-    before(function () {
-      onErrorSpy = sinon.spy()
-      onReadySpy = sinon.spy()
-
-      wrapper = mount(
-        <SingleTextViewerContainer
-          content={subject.content}
-          contentLoadingState={subject.contentLoadingState}
-          error={subject.error}
-          onError={onErrorSpy}
-          onReady={onReadySpy}
-        />)
-    })
-
     it('should render the text subject content', function () {
-      const stv = wrapper.find(SingleTextViewer)
-      expect(stv.prop('content')).to.equal(subject.content)
-    })
-
-    it('should call the onReady prop', function () {
```

Comment:
I'm pretty sure we want to keep this test, because `onReady` tells the task area that we're ready to start a classification.